src/code-templates/.nvmrc

```diff
@@ -1 +1 @@
-16.15.0
+18
```

Comment:
@rluvaton Wouldn't you say that 18.1 (for example) is always more stable than 18?

Reply:
18 will use the latest 18 version you have installed on your system

We can use `lts/<node-18-code-name>` but it will download the newest version every time which is good but exhausting that you need to wait for it to download